api/agent/drivers/driver.go

```diff
@@ -178,6 +181,9 @@ type ContainerTask interface {
 	// UDSDockerDest is the destination mount point for uds path. it is the path
 	// of the directory where the sock file resides inside of the container.
 	UDSDockerDest() string
+
+	// Returns true if network is disabled.
+	DisableNet() bool
```

Comment:
Change to `IsNetworkDisabled()` to be congruent with `IsNetworkReady()`, perhaps?

Reply:
So, here, DisableNet() is orthogonal to driver.IsNetworkReady(). The agent can set this to false regardless of driver.IsNetworkReady(). Currently, agent only sets DisableNet() for status checks, if driver.IsNetworkReady() is false.